app/models/tokite/hook_event/pull_request_review.rb

```diff
@@ -5,16 +5,17 @@ def fields
         {
           event: "pull_request_review",
           repo: hook_params[:repository][:full_name],
-          body: hook_params[:review][:body],
+          body: hook_params[:review][:body] || "",
           user: hook_params[:review][:user][:login],
           review_state: hook_params[:review][:state],
         }
       end
 
       def notify?
-        return false unless hook_params[:action] == "submitted"
-        if hook_params[:review][:state] == "commented"
-          hook_params[:review][:body]
+        if not hook_params[:action] == "submitted" 
```

Comment:
Do not use `not` https://github.com/cookpad/styleguide/blob/master/ruby.ja.md#%E6%BC%94%E7%AE%97%E5%BC%8F

Reply:
Thanks, I fixed it.

Reply:
Sorry, the previous commit contained syntax error. I fixed and modified it.